test/language/class/class-default-constructor-arguments.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class default constructor arguments
+---*/
+var args, self;
```

Comment:
This probably cannot pass in browser environments. How about `that` or `instance` instead of `self`?